object_tracker_0/src/inference.py

```diff
@@ -224,7 +226,11 @@ def do_create_annotations(bboxes):
                 },
                 'iscrowd': 0,
                 'segmentation': []
-            })
+            }
+            score = scores.get(frame_idx, {}).get(obj_id, -1)
+            if score > 0:
+                annotation['attributes']['score'] = float(int(score * 100) / 100)
+            annotations.append(annotation)
```

Comment:
@alejandromarcu did you try uploading the result to CVAT to confirm it's still compatible? It would be good to keep our format compatible so that we can use that tool if we want to

Reply:
I just tried and it worked.